docs/README.android

```diff
@@ -133,21 +113,16 @@ Android platform you first need to setup an Android toolchain using
 the Android NDK which you earlier extracted to <android-ndk>. The
 following commands will create a toolchain suitable for the most
 common scenario.
-The --install-dir option (and therefore the <android-toolchain-arm>/<android-toolchain-x86> value)
-specifies where the resulting toolchain should be installed (your choice).
 
-[NOTICE] Kodi uses the android API Version 21 and gcc version 4.9!
+Building for arm architecture: (This has been tested Feb 4, 2017.)
 
-Building for arm architecture:
-
-    $ cd <android-ndk>
-    $ ls platforms
+    $ cd /$HOME/android-ndk-r12b
     $ cd build/tools
     $ ./make-standalone-toolchain.sh --ndk-dir=../../ \
-      --install-dir=<android-toolchain-arm>/android-21 --platform=android-21 \
+      --install-dir=/$HOME/arm-linux-androideabi-4.9-vanilla/android-21 --platform=android-21 \
```

Comment:
I find it helpful to hard-code the `arm-linux-androideabi-4.9-vanilla` path here. The extra indirection causes way too much scrolling